### DISCUSSION TITLE
Documentation "Use with express session" doesn't work, what's the new way?

### DISCUSSION THREAD

**Discussion by 8o8inCodes** [socketio/socket.io#4364] [Q&A] [answered]
I've been following this guide [use-with-express-session](https://socket.io/how-to/use-with-express-session) and req.session.reload throws an error 
```
Error: failed to load session
    at Immediate._onImmediate (D:\repos\tig\tig-server\node_modules\express-session\session\session.js:94:26)
    at processImmediate (internal/timers.js:466:21)
```
I am using latest node and latest npm packages as of right now.

**Comment by 8o8inCodes** [comment, marked as answer]
My mistake, I needed to make a post request to "/login"

**Reply by darrachequesne** [reply]
Thanks for the update :+1:
